mimic/test/test_swift.py

```diff
@@ -243,6 +289,98 @@ def test_openstack_ids(self):
         self.assertNotIn("/MossoCloudFS_", url)
 
 
+@ddt.ddt
+class SwiftAccountTests(SwiftTestBase):
+    """
+    Swift Account Tests.
+    """
+
+    @ddt.data(
+        (0, 0),
+        (1, 0),
+        (2, 5),
+    )
+    @ddt.unpack
+    def test_head_account(self, container_count, object_count):
+        """
+        HEADing an account correctly reports the container count,
+        object count, and number of bytes used.
+        """
+        # Add containers and objects per parameters
+        for container_index in range(container_count):
+            container_uri = (
+                self.swift_uri +
+                "/container{0}".format(container_index))
+            self.put_container(container_path=container_uri)
+
+            for object_index in range(object_count):
+                object_uri = (
+                    container_uri +
+                    "/object{0}".format(object_index))
+                self.put_object(object_path=object_uri)
+
+        # now try to head the account
+        head_response, head_content = self.head_account()
+
+        total_objects = object_count * container_count
+        total_bytes = total_objects * self.object_size
+
+        # validate the headers
+        self.assertEqual(
+            head_response.headers.getRawHeaders(
+                b"X-Account-Container-Count")[0],
+            "{0}".format(container_count).encode("utf-8"))
+        self.assertEqual(
+            head_response.headers.getRawHeaders(
+                b"X-Account-Object-Count")[0],
+            "{0}".format(total_objects).encode("utf-8"))
+        self.assertEqual(
+            head_response.headers.getRawHeaders(
+                b"X-Account-Bytes-Used")[0],
+            "{0}".format(total_bytes).encode("utf-8"))
+
+    @ddt.data(
+        True,
+        False
+    )
+    def test_post_account_metadata(self, post_metadata):
+        """
+        POSTing an account correctly saves metadata to the account.
```

Comment:
"correctly" is another one of those non-words from https://jml.io/pages/test-docstrings.html that we should avoid.